### ISSUE TITLE
Failure of 'Python Testing Suite - Legacy' Test Cases When Executed Alongside 'Python Testing Suite' Cases

### ISSUE THREAD

**Issue by Survensa** [project-chip/certification-tool#211]
### Describe the bug

Test cases in the "Python Testing Suite - Legacy" fail when executed alongside test cases from both the "Python Testing Suite" and "Python Testing Suite - Legacy." However, there are no issues when multiple test cases are executed within their respective suites, either "Python Testing Suite" or "Python Testing Suite - Legacy."

### Steps to reproduce the behavior

1. Select one test case from Python Testing Suite.
2. Select one test case from Python Testing Suite - Legacy.
3. Run the test case. 

### For example :

![image](https://github.com/project-chip/certification-tool/assets/95559513/9f37ff8a-40f7-4e48-b1e3-cd8c26cc1288)

### Expected behavior

The test case should execute without any issues irrespective of the suite from which it is selected.

### Log files

[UI_Test_Run_2024_03_06_16_07_41.log](https://github.com/project-chip/certification-tool/files/14508767/UI_Test_Run_2024_03_06_16_07_41.log)

### Screenshots

![image](https://github.com/project-chip/certification-tool/assets/95559513/fe31c825-2f79-4991-b9c8-9f6c991667a9)

### Environment

Version: 2.10-beta3+spring2024
Sha: 1fd1429

**Comment by rquidute** [comment]
Hi @Survensa, I performed the execution of TC-ACL-2.2 along TC-SM-1.1 it worked fine.
<img width="590" alt="Screenshot 2024-03-07 at 08 58 09" src="https://github.com/project-chip/certification-tool/assets/116586593/4a977658-fa15-447d-b394-4c30adb0ce79">

Regarding the execution of  TC-ACE-1.2, when I performed the test execution, it is always failing even running alone. I'm still investigating the reason.
